presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeIcebergPositionDeleteQueries.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativeworker;
+
+import com.facebook.presto.testing.QueryRunner;
+import com.facebook.presto.tests.AbstractTestQueryFramework;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.nativeworker.NativeQueryRunnerUtils.createNationWithFormat;
+import static org.testng.Assert.assertEquals;
+
+public abstract class AbstractTestNativeIcebergPositionDeleteQueries
+        extends AbstractTestQueryFramework
+{
+    private final String storageFormat = "PARQUET";
+
+    @Override
+    protected void createTables()
+    {
+        QueryRunner queryRunner = (QueryRunner) getExpectedQueryRunner();
+
+        createNationWithFormat(queryRunner, storageFormat);
+    }
+
+    @Test
+    public void testIcebergReadPositionDeletes()
+    {
+        QueryRunner javaIcebergQueryRunner = (QueryRunner) getExpectedQueryRunner();
+
+        try {
+            javaIcebergQueryRunner.execute("CREATE TABLE iceberg_native_position_delete_test AS SELECT * FROM nation");
+
+            // ASSERT number of rows in the table before delete is executed
+            assertQueryResultCount("SELECT * FROM iceberg_native_position_delete_test", 25);
+
```

Comment:
Can you please add a test to verify there is one row with nationkey = 10 before deleting that value?